pkg/cmd/package/nuget/create/create.go

```diff
@@ -0,0 +1,314 @@
+	outFilePath := pack.BuildOutFileName("nupkg", opts.Id.Value, opts.Version.Value)
+
+	err = pack.BuildPackage(opts.PackageCreateOptions, outFilePath)
+	if err != nil {
+		return err
+	}
+
+	if !opts.NoPrompt {
+		autoCmd := flag.GenerateAutomationCmd(
+			opts.CmdPath,
+			opts.Author,
+			opts.Title,
+			opts.Description,
+			opts.ReleaseNotes,
+			opts.ReleaseNotesFile,
+			opts.Id,
+			opts.Version,
+			opts.BasePath,
+			opts.OutFolder,
+			opts.Include,
+			opts.Verbose,
+			opts.Overwrite,
+		)
+		fmt.Fprintf(opts.Writer, "\nAutomation Command: %s\n", autoCmd)
+	}
+
+	if nuspecFilePath != "" {
+		return os.Remove(nuspecFilePath)
+	}
+
+	return nil
+}
+
+func PromptMissing(opts *NuPkgCreateOptions) error {
+	if len(opts.Author.Value) == 0 {
+		message := "Author"
+		for {
+			var author string
+			if err := opts.Ask(&survey.Input{
+				Message: message,
+				Help:    "Add an author to the package metadata.",
+			}, &author); err != nil {
+				return err
+			}
+			if author == "" {
+				break
+			}
+			opts.Author.Value = append(opts.Author.Value, author)
+			message = message + " (leave blank to continue)"
+		}
+	}
+
+	if len(opts.Author.Value) > 0 {
+		if opts.Title.Value == "" {
+			if err := opts.Ask(&survey.Input{
+				Message: "Nuspec title",
+				Help:    "The title to include in the Nuspec file.",
+			}, &opts.Title.Value); err != nil {
+				return err
+			}
+		}
+
+		if opts.Description.Value == "" {
+			if err := opts.Ask(&survey.Input{
+				Message: "Nuspec description",
+				Help:    "The description to include in the Nuspec file.",
+				Default: "A deployment package created from files on disk.",
+			}, &opts.Description.Value); err != nil {
+				return err
+			}
+		}
+
+		if opts.ReleaseNotes.Value == "" {
+			if err := opts.Ask(&surveyext.OctoEditor{
+				Editor: &survey.Editor{
+					Message: "Nuspec release notes",
+					Help:    "The release notes to include in the Nuspec file.",
+				},
+				Optional: true,
+			}, &opts.ReleaseNotes.Value); err != nil {
+				return err
+			}
+		}
+
+		if opts.ReleaseNotesFile.Value == "" {
+			if err := opts.Ask(&survey.Input{
+				Message: "Nuspec release notes file",
+				Help:    "A path to a release notes file whose contents will be included in the Nuspec file's release notes.",
+			}, &opts.ReleaseNotesFile.Value); err != nil {
+				return err
+			}
+		}
+	}
+
+	return nil
+}
+
+func applyDefaultsToUnspecifiedPackageOptions(opts *NuPkgCreateOptions) error {
+	if opts.Version.Value == "" {
+		opts.Version.Value = pack.BuildTimestampSemVer(time.Now())
+	}
+
+	if opts.BasePath.Value == "" {
+		opts.BasePath.Value = "."
+	}
+
+	if opts.OutFolder.Value == "" {
+		opts.OutFolder.Value = "."
+	}
+
+	if len(opts.Include.Value) == 0 {
+		opts.Include.Value = append(opts.Include.Value, "**")
+	}
+
+	if len(opts.Author.Value) > 0 {
+		if opts.Description.Value == "" {
+			opts.Description.Value = "A deployment package created from files on disk."
```

Comment:
This seems low-information. Maybe just have no description?

If we want to go with some generic placeholder, we could use it for a bit of shameless self-promotion and use "Created by Octopus CLI" as the description :smile:

Reply:
I like the shameless plug idea, will bring it up with the team. The reason this has a default is because it required for a nuspec file.